### PR TITLE
First PR on GitHub for Caption This

### LINKED ISSUE #4: Add a Caption To an Existing Slide
## Can you think of a better caption?
Each slide can have more than one caption. Check out the current slide show and add a caption (or two).
If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by rchan88**
Update caption line.
This PR fixes #4 
# Done
- this is first time
## TODO

### CODE REVIEW

**Comment by rchan88** [comment]
@cristigagea  and @EnsyTehrani Please review my PR

File: _slides/11-rchan88.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hi Renee!
```
Comment (AntheusTest) [inline _slides/11-rchan88.md:7]: '''suggestion
testing suggestion
'''

File: _slides/11-rchan88.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (AntheusTest) [inline _slides/11-rchan88.md:6]: Hi!

**Review by AntheusTest — APPROVED** [review]

**Review by cristigagea — APPROVED** [review]

File: _slides/11-rchan88.md (hunk 2 above)
Comment (EnsyTehrani) [inline _slides/11-rchan88.md:6]: '''suggestion
Hi Renee Chan ;)
'''

Comment (EnsyTehrani) [inline _slides/11-rchan88.md:6]:
```suggestion
Hi Renee Chan ;)
```

**Review by EnsyTehrani — APPROVED** [review]

**Review by cristigagea — APPROVED** [review]

**Pull request merged**